server/routes/api/choropleth.py

```diff
@@ -132,43 +149,40 @@ def choropleth_api():
                                     geojson['coordinates'],
                                     geojson['type']))
             # Process Statistical Observation if valid.
-            if ('data' in stat_var_by_geo.get(geo_id, []) and
-                    (not per_capita or
-                    'data' in population_by_geo.get(geo_id, []))):
+            if ('data' in population_by_geo.get(geo_id, [])):
                 # Grab the latest available data.
-                stat_obs = next(iter(
-                    reversed(stat_var_by_geo[geo_id]['data'].values())))
-
-                # Handle per-capita option.
-                if per_capita:
-                    # Note that this could be from a different period from the
-                    # statistical variable.
-                    # TODO(iancostello): Determine how to handle populations 
-                    # and statistical values from different times.
-                    pop = next(iter(
-                        reversed(population_by_geo[geo_id]['data'].values())))
-                    stat_obs /= pop
-
-                values.append(stat_obs)
-                geo_feature["properties"]["value"] = stat_obs
+                pop = next(iter(
+                    reversed(population_by_geo[geo_id]['data'].values())))
+                geo_feature["properties"]["pop"] = pop
 
             # Add to main dataframe.
             features.append(geo_feature)
-    # Generate plotting information from value list.
-    domain, palette = determine_color_palette(values, per_capita)
 
     # Return as json payload.
     return jsonify({
-        "geoJson": {
-            "type": "FeatureCollection",
-            "features": features
-        },
-        "_PLOTTING_INFO": {
-            "domain": domain,
-            "palette": palette
-        }  
+        "type": "FeatureCollection",
+        "features": features
     }, 200)
 
+def get_sublevel(requested_geoId, display_level):
+    """Returns the best sublevel display for a geoID.
+
+    Args:
+        requested_geoId: The parent geo DCID to find children.
+        display_level: Display level provided or None.
+    Returns:
+       display_level is not None. Otherwise the next sublevel below the parent.
```

Comment:
Feels like an incomplete statement here...

Also, what are acceptable values for display_level? I think you mean place type here.

Reply:
Updated

Reply:
Updated